src/ftsh.c

```diff
@@ -0,0 +1,119 @@
+/***************************************************************************//**
+
+	@file         ftsh.c
+
+	@author       Max Bronckers
+
+	@date         Thursday, 1 May 2018
+
+	@brief        FTSH (Full-Text SHell)
+
+	@compile      make ftsh
+
+*******************************************************************************/
+
+#include "ftsh.h"
+#include "ftsh_functions.h"
+
+/**
+	@brief Parse input into args.
+	@param input The input.
+	@return Null-terminated array of args.
+ */
+char** ftsh_get_input(char *input) {
+	char **args = malloc(8 * sizeof(char *));
```

Comment:
This code appears to be indented using tabs. Please... fix that. In general, please follow the CS233 (Networks) style guide found [here](https://uchicago-cs.github.io/cmsc23300/style_guide.html)